### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by Oliphaunte**
Added routes to app.js, and margins to input boxes

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import {Router, Route, IndexRoute, browserHistory} from 'react-router'
 
 import Title from './components/atoms/Title/Title';
+import Nav from './components/organisms/Nav/Nav';
```
Comment (Oliphaunte) [inline app/app.js]: This is for an example nav bar to show for the router

**Comment by Rulox** [comment]
Hey @Oliphaunte  I think this needs some work, let's work together to find the best solution.

File: app/components/templates/Main/Main.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  constructor(props) {
+      super(props);
+  }
+
+  render() {
+    return(
+      <div className="app">
+        <Nav />
+        
+        <main>
```
Comment (Rulox) [inline app/components/templates/Main/Main.js]: Why main here?

Comment (Oliphaunte) [inline app/components/templates/Main/Main.js]: What do you mean? Like why is it called Main or why do we need it?

Comment (Rulox) [inline app/components/templates/Main/Main.js]: Why do we need it.

Comment (Oliphaunte) [inline app/components/templates/Main/Main.js]: We don't, I just figured an identifier tag for the output would be better than just calling the component.

Comment (Rulox) [inline app/components/templates/Main/Main.js]: I think it's ok if we just put the component to keep it simple, I'd delete the `<main>` tag, as we are not using that kind of tags in the project, just to follow some rules and consistency. It's something we can add in the future . Thoughts? @Oliphaunte

Comment (Rulox) [inline app/components/templates/Main/Main.js]: Also a few considerations in order to merge this. @Oliphaunte 

* I think we can avoid use excessive styling of the navbar, a list of links should be enough for testing purposes. (ie not hover or color effects. In think in that way we make the project much more reusable, if the user has to change too many things that means the project is not a boilerplate but kind of an app)

* I'd add templates and empty components to test the behavior of the router, now we're getting a 404

Comment (Oliphaunte) [inline app/components/templates/Main/Main.js]: That's fine. We can take it out

Yes, I think we should cut it down too. I wanted to throw it in there to test the activeState with the Router, but that can be done for later probably.

**Comment by Rulox** [comment]
Cool ! Thank you for this, can you just remove "react-router" from TODO list? @Oliphaunte

**Pull request merged**